tests/infra/network.py

```diff
@@ -142,13 +146,23 @@ def __init__(
             self.next_node_id = 0
             self.txs = txs
             self.jwt_issuer = jwt_issuer
+            self.service_load = (
+                infra.service_load.ServiceLoad(self) if with_load else None
```

Comment:
It'd be more flexible to pass an optional ServiceLoad than a boolean flag to the network.

Reply:
As discussed, this is now done.